app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -436,49 +485,57 @@ $ curl -i -X PATCH http://kong:8001/plugins/{plugin_id} \
 
 ### Limit query cost by using `config.max_cost`
 
-It's usually a good idea to define a maximum cost applied to any query, regardless
-if the call is within the rate limits for a consumer.
+It's usually a good idea to define a maximum cost applied to any query,
+regardless if the call is within the rate limits for a consumer.
 
 By defining a `max_cost` on our service, we are ensuring no query will run with
 a cost higher than our set `max_cost`. By default it's set to 0, which means
 no limit.
 
 ```
-$ curl -i -X POST http://kong:8001/services/{service}/plugins \
-  --data name=rate-limiting-advanced \
-  --data config.limit=100,10000 \
-  --data config.window_size=60,3600 \
-  --data config.sync_rate=10 \
+$ curl -X POST http://kong:8001/services/{service}/plugins \
+  --data name=graphql-rate-limiting-advanced \
+  --data config.limit=100 \
+  --data config.limit=10000 \
+  --data config.window_size=60 \
+  --data config.window_size=3600 \
+  --data config.sync_rate=0 \
   --data config.max_cost=5000
 ```
 
+If `max_cost` needs to be updated, one can accomplish this by doing the following:
+
 ```
-$ curl -i -X PATCH http://kong:8001/plugins/{plugin_id} \
-  --data config.max_cost=5000
+$ curl -X PATCH http://kong:8001/plugins/{plugin_id} \
+  --data config.max_cost=10000
 ```
 
 ### Using `config.score_factor` to modify costs
 
-GraphQL query cost depend on multiple factors depending on our resolvers
+GraphQL query cost depends on multiple factors depending on our resolvers
```

Comment:
```suggestion
GraphQL query cost depends on multiple factors based on our resolvers
```
Suggest changing the second instance of "depend" in this sentence - "based on", "derived from", etc?